### ISSUE TITLE
[bug] Jasmine tests in describe block can have name collisions

### ISSUE THREAD

**Issue by Frizi** [wallabyjs/public#290]
When there were two tests named differently, but produces the same name when you take `describe` blocks into account, wallaby starts to do funny things.
![wallaby-same-name-bug](https://cloud.githubusercontent.com/assets/919491/10395400/0d6d0e98-6e9d-11e5-9670-52a4c256b720.png)

I can't make these tests both fail at the same time.

In some cases after changing the actual test content, I can force one to stay green when it shouldn't. In not-so-minimal case (my actual test suite that was wrong for a moment), it triggers an endless spinner in bottom right corner with tooltip `Running tests`, which effectively breaks the whole test suite and most of the tests are gray. Worst thing, you have no idea what happened.

Note this issue is not present when there are two `it` blocks with the same name. The `describe` is essential to reproduce this.

```
describe('wallaby', () => {
  it('should work', () => {
    expect(true).toEqual(false);
  });
});

it('wallaby should work', () => {
  expect(true).toEqual(false);
});
```

**Comment by ArtemGovorov** [comment]
Nice find! Fixed in 1.0.37.

**Comment by Frizi** [comment]
Wow, fast reaction and lightning fast release :+1:
